src/cljs/org/broadinstitute/firecloud_ui/page/workspace/analysis/igv.cljs

```diff
@@ -18,6 +18,8 @@
      :tracks (map-indexed (fn [index track]
                             {:name (str "Track " (inc index))
                              :url track
+                             :indexUrl (clojure.string/replace track #".bam" ".bai")
+                             :type "bam"
```

Comment:
I'm not sure if the syntax for replace() in the indexUrl field is correct, but you can guess at what I'm trying to do here ... 

Also, hardcoding type=bam obviously won't work in the long run, but it seems IGV wants this ...